Tests/FluentMongoDriverTests/FluentMongoDriverTests.swift

```diff
@@ -278,6 +282,22 @@ final class FluentMongoDriverTests: XCTestCase {
     }
 }
 
+struct DuplicateIndexMigration: Migration {
+    func prepare(on database: Database) -> EventLoopFuture<Void> {
+        database
+            .schema("index-tests")
+            .field("name", .string, .required)
```

Comment:
```suggestion
            .compositeIdentifier(over: "name", "email")
            .field("name", .string, .required)
```
In light of my remarks above (and since it'll just translate to an additional unique index for Mongo)